keyboards/atreus62/keymaps/xyverz/keymap.c

```diff
@@ -58,67 +66,78 @@ enum atreus52_keycodes {
-    KC_LCTL, KC_LALT, KC_LEFT, KC_RGHT, LOWER,   KC_BSPC, KC_LGUI, KC_ENT,  KC_SPC,  RAISE,   KC_UP,   KC_DOWN, KC_RGUI, KC_RCTL
-  ),
-
-  [_WOW] = LAYOUT( /* Dvorak with minor modifications for playing World of Warcraft */
-    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                            KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSLS,
-    KC_TAB,  KC_QUOT, KC_COMM, KC_DOT,  KC_P,    KC_Y,                            KC_F,    KC_G,    KC_C,    KC_R,    KC_L,    KC_SLSH,
-    KC_CAPS, KC_A,    KC_O,    KC_E,    KC_U,    KC_I,                            KC_D,    KC_H,    KC_T,    KC_N,    KC_S,    KC_MINS,
-    KC_LSFT, KC_SCLN, KC_Q,    KC_J,    KC_K,    KC_X,                            KC_B,    KC_M,    KC_W,    KC_V,    KC_Z,    KC_RSFT,
-    KC_LCTL, KC_LALT, KC_LEFT, KC_RGHT, LOWER,   KC_BSPC, KC_LALT, CTL_T(KC_ENT), KC_SPC,  RAISE,   KC_UP,   KC_DOWN, KC_RGUI, KC_ENT
-  ),
-
-  [_LOWER] = LAYOUT(
-    KC_F11,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,                     KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F12,
-    KC_TILD, KC_GRV,  _______, _______, _______, _______,                   _______, _______, _______, _______, _______, KC_PIPE,
-    _______, _______, _______, _______, _______, _______,                   _______, _______, KC_PLUS, KC_LCBR, KC_RCBR, _______,
-    _______, _______, _______, _______, _______, _______,                   _______, KC_MUTE, KC_VOLD, KC_VOLU, _______, _______,
-    _______, _______, KC_HOME, KC_END,  _______, KC_DEL,  _______, _______, KC_INS,  _______, KC_PGUP, KC_PGDN, _______, _______
-  ),
-  [_RAISE] = LAYOUT(
-    KC_F11,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,                     KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F12,
-    KC_TILD, KC_GRV,  _______, _______, _______, _______,                   _______, _______, _______, _______, _______, KC_BSLS,
-    _______, _______, _______, _______, _______, _______,                   _______, _______, KC_EQL,  KC_LBRC, KC_RBRC, _______,
-    _______, _______, _______, _______, _______, _______,                   _______, KC_MPRV, KC_MPLY, KC_MNXT, _______, _______,
-    _______, _______, KC_HOME, KC_END,  _______, KC_DEL,  _______, _______, KC_INS,  _______, KC_PGUP, KC_PGDN, _______, _______
-  ),
-  [_ADJUST] = LAYOUT(
-    _______, _______, _______, _______, _______, _______,                   _______, _______, _______, _______, _______, _______,
-    _______, RESET,   _______, _______, _______, _______,                   _______, _______, _______, _______, _______, _______,
-    _______, _______, _______, _______, _______, _______,                   _______, QWERTY,  COLEMAK, DVORAK,  _______, WOW,
-    _______, _______, _______, _______, _______, _______,                   _______, _______, _______, _______, _______, _______,
-    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
-  ),
+    [_DVORAK] = { /* dvorak */
```

Comment:
Please revert these to the LAYOUT format.

Reply:
Please elaborate?

Reply:
We've been moving many of the keyboards over to the LAYOUT "macro".    There are a number of reasons to do so.

The main two is heavily customized keymaps thad don't require a bunch of "KC_NO"s in the keymap (like the atreus), and it helps to allow for hardware agnostic keymaps.  

https://qmk.github.io/qmk_firmware/#/feature_layouts